modules/manage/pages/manage-security/configure-saml.adoc

```diff
@@ -0,0 +1,198 @@
+
+[[nameid]]
+=== NameID and User Identity
+The SAML authentication message sent from the IdP to Couchbase Server identify the user using a format called nameID. The nameID format is a colon-delimited string. It can be one of the following values:
+
+* `urn:oasis:names:tc:SAML:2.0:nameid-format:persistent` uses a unique identifier that remains constant across different IdPs and services. This is the default value for Couchbase Server.
+* `urn:oasis:names:tc:SAML:1.1:nameid-format:unspecified` does not set a format for the user identity. It can be any text string. 
+* `urn:oasis:names:tc:SAML:1.1:nameid-format:emailAddress` sets the format to an email address.
+* `urn:oasis:names:tc:SAML:2.0:nameid-format:transient` uses a unique but temporary identifier 
+
+Instead of relying on the nameID, you can have Couchbase Server get the username from an attribute in the SAML message. Use an attribute instead of the nameID if the IdP uses a format that doesn't match Couchbase Server usernames. For example, suppose the IdP only supports identifying users using an email address instead of a username. In this case, you can have the IdP send the user's username in an attribute. Then configure Couchbase Server to extract the username from the attribute instead of using the nameID.
+
+== Configure SAML with the UI
+
+To configure SAML authentication using Web Console:
+
+. In the navigation bar, click *Security*.
+. In the navigation menu, click *SAML*.
+. Turn on the *Enabled* toggle.
+. Complete the following procedures to finish configuring SAML:
+.. <<configure-couchbase-metadata>>
+.. <<configure-the-idp>>
+.. <<configure-single-sign-on>>
+.. Optionally, <<configure-advanced-settings>>. 
+. Click btn:[Save] to save the SAML settings. 
+. Verify that Couchbase Server saved the settings by looking for a green popup stating that the settings were saved. If you do not see this popup, look for error messages at the top of the SAML Configuration page.
+
+After you enable SAML authentication, the top of the configuration page lists several URLs you need to configure your IdP: 
+
+* *Current SP metadata URL*: the URL to retrieve your Couchbase Server's SAML metadata. A link also appears that you can use to download the metadata. Use this link if your IdP is not able to connect to Couchbase Server directly. 
+* *Current SP consume URL*: the URL the IdP sends the user to after they have authenticated.
+* *Current SP logout URL*: the URL to which the IdP sends the user when they log out of the Web Console.
```

Comment:
Logout endpoint that receives and processes SAML logout requests and responses from the identity provider (IdP)